### PR TITLE
Fix new hits for analyzer CA2249

### PR DESCRIPTION
**Pull request by carlossanlop**
Already enabled in dotnet/runtime: https://github.com/dotnet/runtime/blob/08acbb7ad7154e9d18227e4b00e4c9f41af1fffa/eng/CodeAnalysis.src.globalconfig#L635

A new case was added here: https://github.com/dotnet/roslyn-analyzers/pull/6774

cc @dotnet/ncl and @dotnet/area-system-xml , this PR touches code in both your areas.

### CODE REVIEW

File: src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs [hunk 1]
```diff
@@ -279,7 +279,7 @@ protected override PipelineInstruction PipelineCallback(PipelineEntry? entry, Re
             // If we are already logged in and the server returns 530 then
             // the server does not support re-issuing a USER command,
             // tear down the connection and start all over again
-            if (entry.Command.IndexOf("USER", StringComparison.Ordinal) != -1)
+            if (entry.Command.Contains("USER"))
```
Comment (buyaa-n) [inline src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs:282]: Shouldn't we keep the `StringComparison.Ordinal`?

Comment (carlossanlop) [reply src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs:282]: The analyzer suggested this. My assumption is that `Ordinal` is the default comparison when using `Contains`.

Comment (carlossanlop) [reply src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs:282]: @tarekgh is this assumption correct?

**Review by stephentoub — APPROVED** [review]

**Review by tarekgh — APPROVED** [review]

File: src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs (hunk 1 above)
Comment (tarekgh) [reply src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs:282]: String.Contains behavior is doing ordinal comparison by default. We don't need to pass `StringComparison.Ordinal`.

**Pull request merged**